ios/Classes/TiNfcNfcAdapterProxy.m

```diff
@@ -112,6 +170,28 @@ - (void)readerSession:(NFCNDEFReaderSession *)session didInvalidateWithError:(NS
       NO);
 }
 
+#pragma mark NFCReaderSessionDelegate
+
+- (void)tagReaderSession:(NFCTagReaderSession *)session didInvalidateWithError:(NSError *)error
+{
+  [self fireEvent:@"didInvalidateWithError" withObject:@{ @"error" : error.localizedDescription, @"cancelled" : @(error.code == 200) }];
```

Comment:
I think instead of creating new event we should use same approach which is already implemented before. See ```- (void)readerSession:(NFCNDEFReaderSession *)session didInvalidateWithError:(NSError *)error```